src/main/resources/application.properties

```diff
@@ -1,13 +1,7 @@
-spring.application.name=online-book-store
-spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
-spring.datasource.url=jdbc:mysql://localhost:3306/online_book_store?serverTimezone=UTC
-spring.datasource.username=root
-spring.datasource.password=F6Rf@N34,djA3i67
-
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
 spring.jpa.open-in-view=false
 server.servlet.context-path=/api
 
-jwt.expiration=450000
-jwt.secret=HelloitsmeWooow1234216966521321426547657434123155654
+JWT.EXPIRATION=450000
+JWT.SECRET=HelloitsmeWooow1234216966521321426547657434123155654
```

Comment:
why uppercase?

Reply:
I wanted to move the values ​​of these variables into the .env, but it didn't work for me either.